src/applications/hca/tests/unit/components/FormFields/VaMedicalCenter.unit.spec.js

```diff
@@ -182,7 +184,8 @@ describe('hca <VaMedicalCenter>', () => {
         const alert = container.querySelector('va-alert');
 
         expect(options).to.have.lengthOf(mockData.length);
-        expect(options[0]).to.have.attr('value', mockData[0].uniqueId);
+        expect(options[0]).to.have.attr('value', mockData[1].uniqueId);
```

Comment:
I'd probably like a comment or update the `it` text to show that we expect these to be alphabetized?

Reply:
How's that rename for the test look?